apps/site/components/EOL/VulnerabilityChips/index.tsx

```diff
@@ -0,0 +1,41 @@
+import type { FC } from 'react';
+
+import type { Vulnerability } from '#site/types/vulnerabilities';
+
+import VulnerabilityChip from './Chip';
+
+export const SEVERITY_ORDER = ['critical', 'high', 'medium', 'low'] as const;
```

Comment:
Same here, could be reused.

Reply:
addressed via https://github.com/nodejs/nodejs.org/pull/7990/commits/40df1abfcf62c060b0663f95ac24041e15e3b84b